HEADER.js

```diff
@@ -10,7 +10,9 @@ else if (typeof define === 'function' && define.amd) {
 }
 /* _AMD_END_ */
 if (typeof document !== 'undefined' && typeof window !== 'undefined') {
-  if (document instanceof HTMLDocument) {
+  var DocumentConstructor = typeof HTMLDocument !== 'undefined' ? HTMLDocument : Document;
+
+  if (document instanceof DocumentConstructor) {
```

Comment:
```suggestion
  if (document instanceof (typeof HTMLDocument !== 'undefined' ? HTMLDocument : Document)) {
```

Reply:
Another solution is to wrap all the file in a self executed function.

Reply:
Okay, I'll use that ternary operation. Thanks.